src/devices/roccat_elo_7_1_air.c

```diff
@@ -0,0 +1,110 @@
+#include "../device.h"
+
+#include <stdio.h>
+#include <string.h>
+#include <time.h>
+#include <inttypes.h>
+
+static struct device device_elo71Air;
+
+static const uint16_t PRODUCT_ID = 0x3a37;
+
+static int elo71Air_switch_lights(hid_device* hid_device, uint8_t on);
+static int elo71Air_send_inactive_time(hid_device* hid_device, uint8_t num);
+
+void elo71Air_init(struct device** device)
+{
+    device_elo71Air.idVendor = VENDOR_ROCCAT;
+    device_elo71Air.idProductsSupported = &PRODUCT_ID;
+    device_elo71Air.numIdProducts = 2;
```

Comment:
After looking at usage of this parameter at other device's code I think that I did not set it correctly!?

@Sapd Can you please have a look and give a hint, how it is used properly (if necessary)?

Reply:
Yep, it must be 1.
It simply says, how big the array device_elo71Air.idProductsSupported is. As you only provide one id, its size is 1.
When you specify 2, the loop in device_manager.c would run into some other memory and possibly crash or do some other unexpected things.